lib/aasm/persistence/base.rb

```diff
@@ -84,6 +84,11 @@ def state_with_scope(name, *args)
     end
     alias_method :state_without_scope, :state
     alias_method :state, :state_with_scope
+
+  private
+    def should_create_scope?(name, *args)
+      return @state_machine.config.create_scopes && !@klass.respond_to?(name) && (args.first.blank? || args.first[:create_scope] != false)
```

Comment:
I'm still not totally convinced this is the right solution though - other than our weird case where we don't want the 'locked' scope created, I'm trying to think of any other case where anyone would want all the magic scopes created. *except* for certain ones. Especially since this code already checks to see if the method already exists.